src/MessageBird/Objects/SignedRequest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace MessageBird\Objects;
+
+/**
+ * Class SignedRequest
+ *
+ * @package MessageBird\Objects
+ *
+ * @link https://developers.messagebird.com/docs/verify-http-requests
+ */
+class SignedRequest extends Base
+{
+    const BODY_HASH_ALGO = 'sha256';
+
+    /**
+     * The timestamp passed in the MessageBird-Request-Timestamp header of the request.
+     *
+     * @var string
+     */
+    protected $requestTimestamp;
+
+    /**
+     * The request body.
+     *
+     * @var string
+     */
+    protected $body;
+
+    /**
+     * The query parameters for the request.
+     *
+     * @var array
+     */
+    protected $queryParameters = array();
+
+    /**
+     * The signature passed in the MessageBird-Signature header of the request.
+     *
+     * @var string
+     */
+    protected $signature;
+
+    /**
+     * Create a new SignedRequest from PHP globals.
+     *
+     * @return SignedRequest
+     */
+    public static function createFromGlobals()
+    {
+        $body = file_get_contents('php://input');
+        $queryParameters = $_GET;
+        $requestTimestamp = $_SERVER['HTTP_MESSAGEBIRD_REQUEST_TIMESTAMP'];
+        $signature = $_SERVER['HTTP_MESSAGEBIRD_SIGNATURE'];
+
+        $signedRequest = new SignedRequest();
+        $signedRequest->loadFromArray(compact('body', 'queryParameters', 'requestTimestamp', 'signature'));
```

Comment:
What do you think about overriding `loadFromArray` and doing input validation there? Right now, no validation is done here, which can lead to obscure warnings/errors. We also have inconsistency between `createFromGlobals()` and `create()` here.